go/libraries/doltcore/sqle/sqlfmt/schema_fmt.go

```diff
@@ -55,6 +55,10 @@ func FmtColWithNameAndType(indent, nameWidth, typeWidth int, colName, typeStr st
 		}
```

Comment:
We should kill of this whole file, make sure it always goes through the engine. (This was written before the engine supported a bunch of things). Add a TODO / do it?